apps/institution-service/src/controllers/aggregatorController.test.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { Request, Response } from "express";
+import { Aggregator } from "../models/aggregator";
+import { createTestAuthorization } from "../test/utils";
+import { getAggregators } from "./aggregatorController";
+
+describe("getAggregators", () => {
+  it("gets all the aggregators in the database", async () => {
+    const { count, rows: _aggregators } = await Aggregator.findAndCountAll();
+
+    const req = {
+      headers: {
+        authorization: createTestAuthorization({
+          permissions: [],
+        }),
+      },
+    } as Request;
+    let responseData: { aggregators: object[] } = { aggregators: [] };
+    const res = {
+      json: jest
+        .fn()
+        .mockImplementation(
```

Comment:
Are you sure you need to do this mock implementation?

Reply:
It's probably not necessary